include/RAJA/pattern/params/kernel_name.hpp

```diff
@@ -5,24 +5,21 @@
 
 namespace RAJA
 {
-namespace expt
-{
 namespace detail
 {
 
-struct KernelName : public ForallParamBase
+struct Name : public ::RAJA::expt::detail::ForallParamBase
 {
-  RAJA_HOST_DEVICE KernelName() {}
+  RAJA_HOST_DEVICE Name() {}
 
-  KernelName(const char* name_in) : name(name_in) {}
+  Name(const char* name_in) : name(name_in) {}
```

Comment:
Please mark single arg ctors `explicit` to prevent unintended implicit type conversions.